### PR TITLE
Splash page - Update metadata

### PR DESCRIPTION
**Pull request by Ricokola**
Update metadata on Canada.ca splash page according to WET-150

### CODE REVIEW

File: templates/splash/layout-splashpage.html [hunk 1]
```diff
@@ -6,18 +6,19 @@
 		<title>{{ page.title }}</title>
 		<meta content="width=device-width, initial-scale=1" name="viewport"/>
```
Comment (GormFrank) [inline templates/splash/layout-splashpage.html:7]: After further discussions and research, you would need to add the following reference to the Dublin Core namespace right under the `title` tag. What's important here is that it is included before your first instance of a `meta` with `dcterms`. No need for the `profile` attribute on the `head` which would be just for XHTML page and not for HTML.
 tag.
```suggestion
		<link rel="schema.dcterms" href="http://purl.org/dc/terms/" />
		<meta content="width=device-width, initial-scale=1" name="viewport"/>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

File: templates/splash/splashpage.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ title: Canada.ca
 lang: en
 description-en: The Government of Canada website is a single point of access to all programs, services, departments, ministries and organizations of the Government of Canada.
 description-fr: Le site Web du gouvernement du Canada fournit un point d'accès complet à tous les programmes, services, départements, ministères et organismes du gouvernement du Canada.
-creator-en: Government of Canada, Service Canada, Citizen Service Branch, Integrated Channel Management, Web Strategies and Product Management
-creator-fr: Gouvernement du Canada, Service Canada, Direction générale de service aux citoyens, Gestion intégrée des modes de service, Gestion des stratégies et produits Web
-subject-en: Government of Canada, services
-subject-fr: Gouvernement du Canada, services
-dateModified: 2020-06-03
+creator-en: Service Canada
+creator-fr: Service Canada
```
Comment (duboisp) [inline templates/splash/splashpage.md:8]: Change it for "Government of Canada", this splash page is not for Service Canada specifically.

File: templates/splash/splashpage.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ title: Canada.ca
...
+subject-en: GV Government and Politics;Government services
+subject-fr: GV Gouvernement et vie politique;Services gouvernementaux
```
[the remark this answers is not in the document]
Comment (duboisp) [reply templates/splash/splashpage.md:10]: Here the controlled vocabulary used: https://canada.multites.net/cst/index.htm

File: templates/splash/layout-splashpage.html [hunk 4]
```diff
@@ -4,22 +4,24 @@
 	<head>
 		<meta charset="utf-8"/>
 		<title>{{ page.title }}</title>
-		<meta content="width=device-width, initial-scale=1" name="viewport"/>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply templates/splash/layout-splashpage.html:7]: oh, it was moved 2 line below

File: templates/splash/splashpage.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ricokola) [reply templates/splash/splashpage.md:10]: This will need to be discussed for clarifications.

File: templates/splash/splashpage.md [hunk 5]
```diff
@@ -4,11 +4,12 @@ title: Canada.ca
...
 description-fr: Le site Web du gouvernement du Canada fournit un point d'accès complet à tous les programmes, services, départements, ministères et organismes du gouvernement du Canada.
-creator-en: Government of Canada, Service Canada, Citizen Service Branch, Integrated Channel Management, Web Strategies and Product Management
-creator-fr: Gouvernement du Canada, Service Canada, Direction générale de service aux citoyens, Gestion intégrée des modes de service, Gestion des stratégies et produits Web
-subject-en: Government of Canada, services
-subject-fr: Gouvernement du Canada, services
-dateModified: 2020-06-03
+creator-en: Government of Canada
+creator-fr: Gouvernement du Canada
+subject-en: GV Government and Politics;Government services
+subject-fr: GV Gouvernement et vie politique;Services gouvernementaux
```
Comment (duboisp) [inline templates/splash/splashpage.md:10]: Can you remove the second subject "Government services"

**Review by duboisp — CHANGES_REQUESTED** [review]
Pre-approved at the technical review after the second subject is removed,

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**